sdk/search/search-documents/package.json

```diff
@@ -86,6 +94,7 @@
     "events": "^3.0.0"
   },
   "devDependencies": {
+    "autorest": "latest",
```

Comment:
I don't think we usually depend on autorest directly, even as a dev-dep. I know that means folks who regenerate have to install it globally, but I'm a little wary of this.

Reply:
Good point, most contributors will never touch it. Removed!